package.json

```diff
@@ -4,6 +4,9 @@
   "private": true,
   "type": "module",
   "dependencies": {
+    "@emotion/react": "^11.6.0",
+    "@emotion/styled": "^11.6.0",
```

Comment:
You are probably using this package for styled components. You could also use `styled-components` instead.

Reply:
The new material-ui version specifically required this package. Otherwise it doesn't compile